backend/services/UserService.js

```diff
@@ -1,39 +1,49 @@
-const BaseModelService = require('./baseModelService');
+const BaseModelService = require('./BaseModelService');
 
 class UserService extends BaseModelService {
     constructor(bcryptjs) {
         super();
         this.bcryptjs = bcryptjs;
     }
 
-    static get table() {
+    get table() {
         return 'users';
     }
 
-    static register(email, password) {
+    register(email, password) {
```

Comment:
This should be moved to separate handler and the handler should use UserRepository